### ISSUE TITLE
C# Language Mode is no longer an option

### ISSUE THREAD

**Issue by Amarok24** [microsoft/vscode#110252]
Hello,
When I open a .cs file I don't see any syntax highlight for C# anymore.
The Language Mode is set by VSCode to "Plain Text".
When I try to change it, the csharp language mode (and also C and C++) is no longer an option, it just disappeared (I think it happened after I installed and the uninstalled some extensions). Even uninstalling of VSCode from Windows and installing again did not help because Windows stores parts of VSCode configuration somewhere.

I've checked older issues and it already happened here:
https://stackoverflow.com/questions/49755241/php-language-mode-is-no-longer-an-option
https://github.com/Microsoft/vscode/issues/47110
but apparently an update of VSCode solved the issue.

(Sidenote: after uninstalling VSCode again and installing the Insiders version (1.52.0-insider) the issue was gone. But that does not solve the original issue, of course.)

Visual Studio Code version 1.51.0
Windows_NT x64 10.0.19041

Enabled extensions:
csharp | ms- | 1.23.5
(the issue remains also without any extensions)

Thanks,
Jan

![language-mode-bug](https://user-images.githubusercontent.com/7067067/98580267-fc2f7a80-22bf-11eb-953c-2b98f843c4ad.png)

**Comment by aeschli** [comment]
@Amarok24 Can you open the extensions viewlet and search for `@builtin @disabled `?
I suspect the C# and c++ extensions are disabled.

**Comment by Amarok24** [comment]
> @Amarok24 Can you open the extensions viewlet and search for `@builtin @disabled `?
> I suspect the C# and c++ extensions are disabled.

A search for @builtin @disabled didn't show any extensions, but I've then searched just for @builtin and indeed - C/C++ Language Basics and C# Language Basics were in grey color and disabled, so I enabled it and it works now!
I really don't know how that could happen because it used to be enabled.

Thanks for help, much appreciated. I'm closing the issue now.
